Sources/Sentry/include/SentryTraceState.h

```diff
@@ -0,0 +1,96 @@
+#import "SentryId.h"
+#import "SentrySerializable.h"
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class SentryScope, SentryOptions, SentryTracer, SentryUser;
+
+static NSString *const SENTRY_TRACESTATE_HEADER = @"tracestate";
+
+@interface SentryTraceStateUser : NSObject
+
+/**
+ * The id attribute of the user context.
+ */
+@property (nullable, nonatomic, readonly) NSString *userId;
+
+/**
+ * The value of a segment attribute in the user's data bag, if it exists.
+ */
+@property (nullable, nonatomic, readonly) NSString *segment;
+
+/**
+ * Initializes a SentryTraceStateUser with given properties.
+ */
+- (instancetype)initWithUserId:(nullable NSString *)userId segment:(nullable NSString *)segment;
+
+/**
+ * Initializes a SentryTraceStateUser with data from SentryUser.
+ */
+- (instancetype)initWithUser:(nullable SentryUser *)user;
+
+@end
+
+@interface SentryTraceState : NSObject <SentrySerializable>
+
+/**
+ * UUID V4 encoded as a hexadecimal sequence with no dashes (e.g. 771a43a4192642f0b136d5159a501700)
+ * that is a sequence of 32 hexadecimal digits.
+ */
+@property (nonatomic, readonly) SentryId *traceId;
+
+/**
+ * Public key from the DSN used by the SDK.
+ */
+@property (nonatomic, readonly) NSString *publicKey;
+
+/**
+ * The release name as specified in client options, usually: package@x.y.z+build.
+ */
+@property (nullable, nonatomic, readonly) NSString *releaseName;
```

Comment:
`m`: Why not `release`?
```suggestion
@property (nullable, nonatomic, readonly) NSString *release;
```

Reply:
I saw you resolving this, but it stayed the same. Can you explain why, please?

Reply:
I changed it, and have to changed it back, ARC does not allow a property to be called release. Thats why SentryOptions also use releaseName